definitions/managed sevices.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Managed sevices
+status: Feedback Appreciated
+category: property
+---
+## Managed sevices
+### What it is
+ A complete safety net to your technical environment. It is the trust that you can have in a team to look after, both proactively and reactively, an ever-changing database environment. Above all, it is the knowledge that you have a team of experts to cover all areas of issues!
+
+Listed below are the main challenges Managed Services addresses and resolves.
+
+### Problem it addresses
+
```

Comment:
This whole section needs to be compressed. No section should be more than 3-4 sentences. Look at the commonalities between these benefits and try again. I want to re-emphasize that your writing style and enthusiasm is wonderful and I would love to get your definition into the next version of the glossary.

Reply:
read